src/main/resources/io/viash/platforms/nextflow/WorkflowHelper.nf

```diff
@@ -6,10 +6,33 @@ import java.util.regex.Pattern
 import java.io.BufferedReader
 import java.io.FileReader
 import java.nio.file.Paths
+import java.nio.file.Files
 import groovy.json.JsonSlurper
 import groovy.text.SimpleTemplateEngine
 import org.yaml.snakeyaml.Yaml
 
+// Recurse upwards until we find a '.build.yaml' file
+def findBuildYamlFile(path) {
+  child = path.resolve(".build.yaml")
```

Comment:
Use `def` inside functions to ensure you're not overwriting global variables >_>

```suggestion
  def child = path.resolve(".build.yaml")
```